cloud/clouds.go

```diff
@@ -156,6 +156,7 @@ type cloudSet struct {
 
 // cloud is equivalent to Cloud, for marshalling and unmarshalling.
 type cloud struct {
+	Name             string                 `yaml:"name"`
```

Comment:
Also, I'll add a test to show that we ignore the name field in the YAML file if it's there.